.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
did you mean to commit the files in the .vs folder?

Reply:
This is my Visual Studio Explore show all files. Even after changing the .gitignore file.

![image](https://user-images.githubusercontent.com/67446778/156072556-e241a8f0-6c58-444b-9482-0af838208f0f.png)


This is the contents of the .gitignore file:

_site
.jekyll-cache
.vs
.vs\VSWorkspaceState.json
.vs\slnx.sqlite
.vs\ProjectSettings.json
.vs\docs
.vs\docs\v17
.vs\docs\v17\.suo

![image](https://user-images.githubusercontent.com/67446778/156072223-0afde5d9-d157-4d40-85fb-c50e0821f016.png)


I cllosed Visual Studio, Opened Visual Studio and yet as soon as I changed/updated the `settings-and-options.md they show up again as changed.

![image](https://user-images.githubusercontent.com/67446778/156071909-cdf40570-eece-4261-b145-76cdf0bc3421.png)